ocs_ci/ocs/dr/dr_workload.py

```
@@ -2193,6 +2286,12 @@ def delete_workload(
                 f"oc delete -k {self.workload_path} -n {self.workload_namespace}",
                 ignore_error=True,
             )
+            if self.recipe_enabled:
+                log.info(f"Deleting recipe from {cluster.ENV_DATA['cluster_name']}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)